src/contracts/deposits-registry/sanity-checker/integrity-checker/integrity-checker.service.ts

```diff
@@ -75,8 +72,8 @@ export class DepositIntegrityCheckerService {
    * @param {number} blockNumber - Block number to check the deposit root against.
    * @returns {Promise<void>} A promise that resolves if the roots match, otherwise throws an error.
    */
-  public async checkFinalizedRoot(blockNumber: number): Promise<void> {
-    return this.checkRoot(blockNumber, this.finalizedTree);
+  public async checkFinalizedRoot(tag: string | number): Promise<boolean> {
```

Comment:
method's doc contains  blockNumber 
tag -> blockTag
blockNumber in doc -> blockTag